content/millennium/r4/general-clinical/condition.md

```diff
@@ -164,10 +164,13 @@ Create a new Condition.
 
 _Implementation Notes_
 
-* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* The Condition Create API supports only the API fields mentioned below. Unsupported fields will be ignored.
 * Modifier fields should not be provided, and will cause the transaction to fail.
-* The code.coding field can have at most 2 codings, one of which must be set as userSelected true and the other one must be set as userSelected false.
-* Currently `problem-list-item` and `encounter-diagnosis` are supported.
+* The code.coding field can have at most 2 codings.
+  * One of the codings must have userSelected: true
+  * The other coding must have userSelected: false
+* The Condition Create API currently supports only Conditions with a category of `problem-list-item` or `encounter-diagnosis`.
+* An `encounter-diagnosis` Condition will be automatically prioritized to the least significant priority by the Cerner Millennium EHR.
```

Comment:
Are we updating this for DSTU2 as well? AC: Update fhir.cerner.com R4 and DSTU2 Condition Create Implementation Notes by adding...

Reply:
Yes, we should update DSTU2 as well

Reply:
https://github.com/cerner/fhir.cerner.com/pull/602/commits/f6310eebfe043f0f9b9f9eb43cff9c65ca7a9d4a